docs/docs-content/palette-cli/commands/pde.md

```diff
@@ -0,0 +1,422 @@
+
+
+## Registry
+
+Use the `registry` subcommand to manage Palette Registries for App Mode. The `registry` subcommand supports the following subcommands.
+
+  * `create` - Create a Palette Registry.
+
+  * `delete` - Delete a Palette Registry.
+
+  * `list`   - List Palette Registries. 
+
+
+  ### Create
+
+  The `create` subcommand creates a Palette Registry. You can specify an Open Container Initiative (OCI) or a Helm registry. Use the subcommand `oci` or `helm` to specify the type of registry you want to create. The `create` subcommand supports the following subcommands. 
+
+  * `oci` - Create an OCI registry. Use the following flags to create an OCI registry.
+
+  * `helm` - Create a Helm registry. Use the following flags to create a Helm registry.
+
+  The following flags are supported by the `oci` and `helm` subcommands.
+  
+  | **Flag**              | **Description**                                                              | **Type**    |
+  |-------------------|--------------------------------------------------------------------------|---------|
+  | `--endpoint` | The endpoint of the registry. | string |
+  | `--help` | Help for the `oci` or the `helm` subcommand. | - |
+  | `--username` | The username of the OCI registry. | string |
+  | `--password` | The password of the OCI registry. | string |
+
+  The following example shows how to create an OCI registry.
+
+  ```shell hideClipboard
+  palette pde registry create oci --endpoint https://harbor.internal.abc.org  \
+  --name internal-packs --username harbor --password *********
+  ```
+
+
+
+  ### List
+
+  The `list` subcommand lists all Palette Registries availble in App Mode. The `list` subcommand supports the following flags.
+
+  | **Flag**  | **Description**      | **Type**    |
+  |-----------|----------------------|---------|
+  | `--type` | The type of registry. Allowed values are `oci` and `helm`. | string |
+  | `--help` | Help for the `list` subcommand. | - |
+
+
+  ### Delete
+
+  The `delete` subcommand deletes a Palette Registry from App Mode. You can specify an Open Container Initiative (OCI) or a Helm registry. Use the type flag `--type` to specify the type of registry you want to delete. The `delete` subcommand supports the following flags.
+
+  | **Flag**              | **Description**                                                              | **Type**    |
+  |-------------------|--------------------------------------------------------------------------|---------|
+  | `--name` | The name of the registry. | string |
+  | `--type` | The type of registry. Allowed values are `oci` and `helm`. | string |
+  | `--help` | Help for the `delete` subcommand. | - |
+
+## Virtual Cluster
+
+You can use the `virtual-cluster` subcommand to manage Palette Virtual Clusters. Below is a list of the supported subcommands you can use. Use the `--help` flag to learn more about each subcommand.
+
+- `create` - Create a Palette Virtual Cluster.
+
```

Comment:
It seems occurrences that follow this initial "Palette Virtual Cluster" should be "virtual cluster" to follow our internal style.